### ISSUE TITLE
IMDB Watchlist

### ISSUE THREAD

**Issue by deepika05** [barbequesauce/Watcher3#84]
**Describe the bug**
I loaded up the IMDB watchlist as shown here -> https://github.com/nosmokingbandit/watcher/wiki/Syncing-Watch-Lists
But i'm getting a warning in Watcher log and no items from my watchlist gets loaded into Watcher

"WARNING [2019-11-29 05:21:36,780] core.rss.imdb.sync.35: http://rss.imdb.com/user/ur*******/watchlist does not look like a valid imdb list"

I tried this with https as well; still the same

**Comment by barbequesauce** [comment]
I can reproduce.

Doing some investigation, it seems IMDb watchlists are deliberately broken unless you have a paid Pro membership. Can anyone confirm that their IMDB watchlists are working without that membership? @ellnic @scambra 

Alternatively, @deepika05 I’m using Trakt watchlists just fine..

**Comment by deepika05** [comment]
Thanks barbequesauce.. but i'm getting this with a Trakt public watchlist. I confirmed that i can view the items on the watchlist without logging in to Trakt. No items show up in Library after this sync
`INFO [2019-11-29 17:05:46,099] core.ajax.get_log_text.569: Dumping log file log.txt to text.
INFO [2019-11-29 17:05:31,395] CPTaskScheduler._task.273: == Finished Task: Trakt Sync ==
INFO [2019-11-29 17:05:31,167] core.trakt.sync.50: Found 0 new movies from Trakt lists.
INFO [2019-11-29 17:05:31,164] core.trakt.sync_rss.137: Trakt RSS sync complete.
INFO [2019-11-29 17:05:30,935] core.trakt.sync_rss.131: Storing last synced date.
xml.etree.ElementTree.ParseError: not well-formed (invalid token): line 10, column 1470
File "<string>", line None
parser.feed(text)
File "/usr/lib/python3.6/xml/etree/ElementTree.py", line 1314, in XML
root = ET.fromstring(feed)
File "/opt/watcher3/core/trakt.py", line 89, in sync_rss
Traceback (most recent call last):
ERROR [2019-11-29 17:05:30,864] core.trakt.sync_rss.91: Trakt rss request.
INFO [2019-11-29 17:05:30,043] core.trakt.sync_rss.85: Syncing Trakt RSS watchlist watchlist. Last sync: 2000-01-01 00:00:00
INFO [2019-11-29 17:05:30,001] core.trakt.sync.30: Syncing Trakt lists.
INFO [2019-11-29 17:05:30,000] CPTaskScheduler._task.248: == Executing Task: Trakt Sync Per User Command ==`
